cmd/jaeger/internal/extension/jaegerquery/factory_test.go

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2024 The Jaeger Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package jaegerquery
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/extension"
+)
+
+func Test_NewFactory(t *testing.T) {
+	factory := NewFactory()
+	assert.Equal(t, factory.Type(), component.Type("jaeger_query"))
+	assert.Equal(t, factory.CreateDefaultConfig(), createDefaultConfig())
+}
+
+func Test_CreateDefaultConfig(t *testing.T) {
+	defaultConfig := createDefaultConfig()
+	assert.NotEmpty(t, defaultConfig)
+}
+
+func Test_CreateExtension(t *testing.T) {
+	set := extension.CreateSettings{
+		ID: component.NewID(component.Type("jaeger_query")),
+	}
+	defaultConfig := createDefaultConfig()
```

Comment:
```suggestion
	cfg := createDefaultConfig()
```
it's not relevant to var name that it's "default"